README.md

```diff
@@ -1,31 +1,133 @@
+    Library](https://docs.python.org/3/library/). These should be drop-in
+    replacements for the corresponding Python modules, although many have
+    reduced functionality or missing methods or classes (which may not be an
+    issue for many most cases).
 
- * **python-ecosys**: Compatible, but reduced-functionality versions of modules from the larger Python ecosystem, for example that might be found in the [Python Package Index](https://pypi.org/).
+ * **python-ecosys**: Compatible, but reduced-functionality versions of
+     packages from the larger Python ecosystem, for example that might be
+     found in the[Python Package Index](https://pypi.org/).
 
-* **micropython**: MicroPython-specific modules that do not have equivalents in other Python environments. These are typically hardware drivers or highly-optimised alternative implementations of functionality available in other Python modules.
+ * **micropython**: MicroPython-specific packages that do not have equivalents
+     in other Python environments. This includes drivers for hardware
+     (e.g. sensors, peripherals, or displays), or libraries to work with
+     embedded functionality (e.g. bluetooth), or MicroPython-specific
+     packages that do not have equivalents in CPython.
 
- * **unix-ffi**: These modules are specifically for the MicroPython Unix port and provide access to operating-system and third-party libraries via FFI.
+* **unix-ffi**: These packages are specifically for the MicroPython Unix port
+    and provide access to operating-system and third-party libraries via FFI,
+    or functionality that is not useful for non-Unix ports.
 
-Usage
------
+## Usage
 
-Many libraries are self contained modules, and you can quickly get started by
-copying the relevant Python file to your device. For example, to add the
-`base64` library, you can directly copy `python-stdlib/base64/base64.py` to the `lib`
-directory on your device.
+To install a micropython-lib package, there are four main options. For more
+information see the [Package management documentation](https://docs.micropython.org/en/latest/reference/packages.html)
+documentation.
 
-Other libraries are packages, in which case you'll need to copy the directory instead. For example, to add `collections.defaultdict`, copy `collections/collections/__init__.py` and `collections.defaultdict/collections/defaultdict.py` to a directory named `lib/collections` on your device.
+### On a network-enabled device:
 
-Future plans (and new contributor ideas)
-----------------------------------------
+As of MicroPython v1.20 (and nightly builds since October 2022), boards
+with WiFi and Ethernet support include the `mip` package manager.
+
+```py
+>>> import mip
+>>> mip.install("package-name")
+```
+
+### Using `mpremote` from your PC:
+
+This is the officially-supported tool for interacting with a MicroPython
+device, and since v0.4.0 it supports installing packages using the `mip`
+command.
```

Comment:
```suggestion
`mpremote` is the officially-supported tool for interacting with a MicroPython
device and, since v0.4.0, support for installing packages is provided by using the `mip`
command.
```